components/DataDocs/SwaggerDocs.jsx

```diff
@@ -0,0 +1,110 @@
+import {
+  Accordion,
+  AccordionCollapse,
+  useAccordionToggle,
+} from "react-bootstrap";
+import ReactMarkdown from "react-markdown";
+import useRequest from "../../utils/useRequest";
+
+function CustomToggle({ children, eventKey }) {
+  const decoratedOnClick = useAccordionToggle(eventKey, () =>
+    console.log("totally custom!")
+  );
+
+  return (
+    <div
+      className="row toggle align-items-center"
+      type="div"
+      onClick={decoratedOnClick}
+    >
+      {children}
+    </div>
+  );
+}
+const SwaggerDocs = () => {
+  const { data, error } = useRequest(
+    "https://clean-swagger-inunbrtacq-uk.a.run.app"
+  );
+  // console.log("Swagger:\n", data);
+  const order = [
+    "covid_us",
+    "covid_historical",
+    "demographics",
+    "economic_snapshots",
+    "economics",
+    "mobility_devices",
+    "mobility_locations",
+    "hhs",
+    "us_counties",
+    "us_states",
+    "npi_us",
+    "covid_global",
+    "covidtrackingproject",
+    "nytimes_covid",
+    "usafacts_covid",
+  ];
+
+  return (
+    <div className="swagger-docs container pb-100">
+      <h2 className="pt-50">Datasets</h2>
+      <div className="row header">
+        <div className="col-md-3 col-12 dataset">Dataset</div>
+        <div className="col-md-3 col-12 table-name">Table name</div>
+        <div className="col-md-6 col-12 summary">Summary</div>
+      </div>
+      {data &&
+        Object.keys(data.definitions).map((k, i) => {
```

Comment:
This `k` looks unused. Should we instead change this to a `order.map` call instead of `Object.keys(data.definitions).map`?